doc_user/UserManual.adoc

```diff
@@ -193,9 +193,9 @@ The second and third list still contains all diseases.
  +
  +
 _Example: Disease Types can be defined by the Administrator. Ask your Administrator how to do it or see
-<<general-data,General Data>> in this document._
+<<settings,Settings>> in this document._
  +
-* *Diagnosis.* A maximum of three diseases can be diagnosed per each attendance (“new attendance” or “re-attendance”
+* *Diagnosis.* A maximum of three diseases can be diagnosed per each attendance (“new attendance” or “readmittance”
```

Comment:
Here the term "Re-Attendance" is the one commonly used to refer to OPD cases: it indicates a subsequent (visit) concerning the same medical case (same disease) or a planned followup.

Reply:
Readmittance is a more appropriate English word for "re-attendance" (which BTW is only correct without the "hyphen") and I have been changing it in the resource bundle files.

Reply:
done